app/retail/templates/shared/bottom_notification.html

```diff
@@ -17,5 +17,5 @@
 {% endcomment %}
 {% load i18n %}
 <div class="row bottom_notification font-caption">
-  {% blocktrans %}<strong>2018 June 8</strong> - Have you seen our  <a style="margin-right: 5px;" href="https://medium.com/gitcoin/tutorial-how-to-price-work-on-gitcoin-49bafcdd201e" target="_blank" rel="noopener noreferrer">Tutorial on How to Price Work on Gitcoin</a>?{% endblocktrans %}
+  {% blocktrans %}<strong>2018 June 15</strong> -{% trans "Browse funded work in OSS on the go.  Check out the " %} <a style="margin-right: 5px;" href="https://gitcoin.co/ios">Gitcoin iOS app</a>!{% endblocktrans %}
```

Comment:
> django.template.exceptions.TemplateSyntaxError: 'blocktrans' doesn't allow other block tags (seen 'trans "Browse funded work in OSS on the go.  Check out the "') inside it

@thelostone-mc

Reply:
derp derp! happened during the rebase 
Fixed